vadr/1.4.2/Dockerfile

```diff
@@ -0,0 +1,102 @@
+RUN apt-get update && apt-get install -y \
+ wget \
+ perl \
+ curl \
+ unzip \
+ build-essential \
+ autoconf \
+ libinline-c-perl \
+ liblwp-protocol-https-perl && \
+ apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# install VADR
+# download entire VADR source code from GitHub release
+# use vadr-install.sh script to install VADR into $VADRINSTALLDIR (set to /opt/vadr)
+# this script grabs files from tagged release and sets things up in /opt/vadr/vadr
+# last step is to delete the original source code that is a duplicate (/opt/vadr/vadr-$VADR_VERSION)
+RUN mkdir -p ${VADRINSTALLDIR} && \
+ cd ${VADRINSTALLDIR} && \ 
+ wget https://github.com/ncbi/vadr/archive/refs/tags/vadr-${VADR_VERSION}.tar.gz && \
+ mkdir vadr-${VADR_VERSION} && tar -xzf vadr-${VADR_VERSION}.tar.gz -C vadr-${VADR_VERSION} --strip-components 1 && \
+ rm vadr-${VADR_VERSION}.tar.gz && \
+ bash vadr-${VADR_VERSION}/vadr-install.sh linux && \
+ rm -rf vadr-${VADR_VERSION}/ && \
+ mkdir /data
+
+# install the latest sarscov2 models 
+# copy calici models into VADRMODELDIR to allow VADR tests to pass completely
+# cleanup duplicate copies of model files
+RUN wget -O vadr-models-sarscov2.tar.gz https://ftp.ncbi.nlm.nih.gov/pub/nawrocki/vadr-models/sarscov2/${VADR_SARSCOV2_MODELS_VERSION}/vadr-models-sarscov2-${VADR_SARSCOV2_MODELS_VERSION}.tar.gz && \
+ tar -xf vadr-models-sarscov2.tar.gz && \
+ mkdir -vp ${VADRMODELDIR} && \
+ cp -nv /vadr-models-sarscov2-${VADR_SARSCOV2_MODELS_VERSION}/* ${VADRMODELDIR} && \
+ rm -rf /vadr-models-sarscov2* && \
+ cp -nv ${VADRINSTALLDIR}/vadr-models-calici/* ${VADRMODELDIR} && \
+ rm -rf ${VADRINSTALLDIR}/vadr-models-calici/
+
+# Virus model files other than sarscov2 will need to be made available to vadr either in
+# the $VADRMODELDIR or another path can be specified using the 'v-annotate.pl -mdir' option.
+# These files will need to be mounted into the container at runtime, e.g. 'docker run -v' option.
+
+# set working directory
+WORKDIR /data
+
+FROM app as test 
+
+# download B.1.1.7 genome from Utah
+ADD https://raw.githubusercontent.com/StaPH-B/docker-builds/master/tests/SARS-CoV-2/SRR13957123.consensus.fa /test-data/SRR13957123.consensus.fa
+
+# print help options (which prints version at top)
+# run test script included w VADR
+# test terminal N trimming script
+# run v-annotate.pl on trimmed B.1.1.7 genome
+RUN v-annotate.pl -h && \
+ /opt/vadr/vadr/testfiles/do-install-tests-local.sh && \ 
+ /opt/vadr/vadr/miniscripts/fasta-trim-terminal-ambigs.pl \
+        /test-data/SRR13957123.consensus.fa \
+        --minlen 50 \
+        --maxlen 30000 \
+        > /test-data/SRR13957123.consensus.trimmed.fasta && \
+ v-annotate.pl --noseqnamemax --glsearch -s -r --nomisc \
+       --mkey sarscov2 --lowsim5seq 6 --lowsim3seq 6 --alt_fail lowscore,insertnn,deletinn \
+       "/test-data/SRR13957123.consensus.trimmed.fasta" \
+       "SRR13957123-vadr-outdir" && \
+  ls SRR13957123-vadr-outdir
```

Comment:
This test 
1. downloads the consensus fasta from this github repo
2. runs the built-in vadr tests
3. runs the vadr workflow on the downloaded fasta

Then shows off the final directory